app/Http/Controllers/HomeController.php

```diff
@@ -101,4 +106,26 @@ public function search(SearchRequest $request)
 
         return view('search', $data);
     }
+
+    public function getPhone($phone)
+    {
+        $pattern = '/^(\+84|0)\d{9,10}$/';
+
+        if (preg_match($pattern, $phone)) {
+            $id = Auth::id();
+            $data = [
+                'phone' => $phone,
+            ];
+            $this->user->update($id, $data);
+
+            return response()->json([
+                'data' => '1',
+            ]);
+        } else {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.ControlStructureSpacing.SpacingAfterOpen</br>**Severity**: error</br>**File**: app/Http/Controllers/HomeController.php</br>**Line**: 124</br>Blank line found at start of control structure